assembl/static2/js/app/utils/csrf.js

```diff
@@ -0,0 +1,21 @@
+import request from 'superagent';
+import urljoin from 'url-join';
+
+export const basePath = () => {
+  const useStargate = document.getElementById('useStargate') ? document.getElementById('useStargate').value : null;
+
+  let path = `${window.location.protocol}//${window.location.host}`;
+
+  if (useStargate === 'true') {
+    path = `${window.location.protocol}//localhost:3000`;
+  }
+
+  return path;
```

Comment:
basePath is not related to csrf, but to the use of stargate or not. Should it be in a separated file, like utils/stargate.js?